src/javascript/app/base/client.js

```diff
@@ -137,7 +137,7 @@ const Client = (() => {
                 redirection(response);
             });
         } else {
-            redirection(response);
+            setTimeout(() => {  redirection(response); }, 500);
```

Comment:
For non-binary domains, adding a set timeout before redirection

Reply:
@amina-deriv would you please add a comment in the code as well.